### ISSUE TITLE
Confluent Control Center shows connector as Degraded but validate script gives green light

### ISSUE THREAD

**Issue by rmoff** [confluentinc/cp-demo#232]
cp-demo 5.5.1, started up from scratch using `./scripts/start.sh`

Confluent Control Center: 
![image](https://user-images.githubusercontent.com/3671582/86344362-1a220a00-bc52-11ea-82e0-20d651ebb95c.png)

![image](https://user-images.githubusercontent.com/3671582/86344449-345be800-bc52-11ea-9158-54ee06f7e4e8.png)

Validate script: 

```
➜ ./validate_connectors_running.sh
Connector wikipedia-irc is in RUNNING state
Connector replicate-topic is in RUNNING state
Connector elasticsearch-ksqldb is in RUNNING state
```

## Suggestion

Validate script should also check task status using the `.tasks[].state` return of the `status` API

**Comment by ybyzek** [comment]
Agree, a connector with a failed task should not pass `./validate_connectors_running.sh`.  Suggestion to check task state is excellent.

The screenshot shown above demonstrates a separate & unrelated issue for C3 that a connector with 1 task that is failed should not show "degraded", it should show "failed" (this is tracked in MMA-7995)
